desktop/libs/notebook/src/notebook/connectors/spark_shell.py

```diff
@@ -42,7 +42,8 @@
 
 
 class SparkConfiguration(object):
-
+  from spark.conf import LIVY_DEFAULT_QUEUE, LIVY_DRIVER_MEMORY, LIVY_DRIVER_CORES, LIVY_EXECUTOR_MEMORY, LIVY_EXECUTOR_CORES
+  from notebook.conf import NOTEBOOK_LIVY_DEFAULT_QUEUE, NOTEBOOK_LIVY_DRIVER_MEMORY, NOTEBOOK_LIVY_DRIVER_CORES, NOTEBOOK_LIVY_EXECUTOR_MEMORY, NOTEBOOK_LIVY_EXECUTOR_CORES
```

Comment:
Is it really needed to have Livy specific settings in notebook?

To me, it seems clearer to just keep them in app/spark (so only changing spark.conf, not notebook.conf) and keep notebook language agnostic, especially all the logic would be contained in the spark_shell.py connector.

Except this, LGTM

Reply:
so the first is to set defaults for all pyspark sessions, the second is for notebooks which may have a different queue.

Reply:
Sorry if dumb question: except in Notebook, where else in Hue a user can use PySpark?

Reply:
So I went back and tested it and it looks like i create a weird race condition on myself (as it would bounce between 2 and 4 gb).  I'll revert it to just the livy overrides.